docs/data/osdf/install-cache-rpm.md

```diff
@@ -0,0 +1,125 @@
+title: Installing the OSDF Cache by RPM
+
+Installing the OSDF Cache
+=========================
+
+This document describes how to install a Pelican-based Open Science Data Federation (OSDF) Cache service via RPMs.
+This service allows a site or regional network to cache data 
+
+!!! note
+    The cache must be registered with the OSG prior to joining the data federation.
+    You may start the registration process prior to finishing the installation by [using this link](#registering-the-cache) 
+    along with information like:
+
+    * Resource name and hostname
+    * Administrative and security contact(s)
+
+
+Before Starting
```

Comment:
I made some changes to the origin doc in this section that we should probably reflect here: https://github.com/brianhlin/docs/blob/pr/pelican-osdf-origin.SOFTWARE-5962/docs/data/osdf/install-origin-rpm.md